ckanext/dhis2harvester/templates/source/refresh_dhis2_credentials.html

```diff
@@ -7,6 +7,11 @@
 <H2>
   {{_('Refresh DHIS2 Harvester')}}
 </H2>
+
+<div class="alert alert-warning">
+  <p>Please confirm your DHIS2 login details</p>
```

Comment:
This needs to be a translated string {{_('Message')}}.
Could you add a full stop at the end?
Could you reword "Refresh DHIS2 Harvester" to "Refresh DHIS2 Source" as well? Unless it's covered in the other PR about that thing?